### PR TITLE
TDR-2523 - Show password hints on the reset password page

### PR DESCRIPTION
**Pull request by vimleshtna**

### CODE REVIEW

File: themes/tdr/login/messages/messages_en.properties [hunk 1]
```diff
@@ -23,7 +23,11 @@ loggedInTitle=The National Archives Transfer Digital Records
 mainHeader=Transfer Digital Records
 passwordNew=New password
 passwordReset=Password reset
-passwordRestrictions=Password must be at least 8 characters
+passwordInfo=Enter a new password below.
+passwordRestrictionsInfo=Your password must have:
```
Comment (TomJKing) [inline themes/tdr/login/messages/messages_en.properties:27]: Maybe rename these a bit to make it clear what they are, it'll just make it clearer in the template what the messages are, something like:

- `passwordRestrictionCharacters`
- `passwordRestrictionCapitalLetter`
- `passwordRestrictionNumber`

**Review by TomJKing — COMMENTED** [review]
Looks good. Thanks. Just one minor comment.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks. Once its deployed onto intg, could you do a manual test just double check it look OK.

**Comment by vimleshtna** [comment]
> Once its deployed onto intg, could you do a manual test just double check it look

Yeah, sure.

**Pull request merged**